monoloco/visuals/webcam.py

```diff
@@ -0,0 +1,204 @@
+# pylint: disable=W0212
+"""
+Webcam demo application
+
+Implementation adapted from https://github.com/vita-epfl/openpifpaf/blob/master/openpifpaf/webcam.py
+
+"""
+
+import time
+import logging
+
+import torch
+import matplotlib.pyplot as plt
+from PIL import Image
+try:
+    import cv2
+except ImportError:
+    cv2 = None
+
+from openpifpaf import decoder, network, visualizer, show, logger
+import openpifpaf.datasets as datasets
+from openpifpaf.predict import processor_factory, preprocess_factory
+
+from ..visuals import Printer
+from ..network import Loco
+from ..network.process import preprocess_pifpaf, factory_for_gt
+from ..predict import download_checkpoints
+
+LOG = logging.getLogger(__name__)
+
+def factory_from_args(args):
+
+    # Model
+    dic_models = download_checkpoints(args)
+    args.checkpoint = dic_models['keypoints']
+
+    logger.configure(args, LOG)  # logger first
+
+    if args.output_types is None:
+        args.output_types = ['multi']
+
+    assert 'bird' not in args.output_types
```

Comment:
This is a bit confusing for me, can you please directly assert what output types do you expect?